Try less samples of anti-aliasing: this code works on all platforms except RPI4 which crashes on 8 with a segfault.
Perhaps due to a bug in SDL2 or Mesa3d, need to restest when bumping Buildroot.

Signed-off-by: Laurent Merckx <laurent-merckx@skynet.be>

--- a/backends/platform/sdl/sdl.cpp	2020-10-24 14:14:16.284656973 +0200
+++ b/backends/platform/sdl/sdl.cpp	2020-10-24 14:14:50.532367158 +0200
@@ -346,7 +346,7 @@ void OSystem_SDL::detectAntiAliasingSupp
 	_capabilities.openGLAntiAliasLevels.clear();
 
 	int requestedSamples = 2;
-	while (requestedSamples <= 32) {
+	while (requestedSamples <= 4) {
 		SDL_GL_SetAttribute(SDL_GL_MULTISAMPLEBUFFERS, 1);
 		SDL_GL_SetAttribute(SDL_GL_MULTISAMPLESAMPLES, requestedSamples);
 
